gym_snake/envs/snakeGameGym.py

```diff
@@ -179,3 +185,13 @@ def check_body_collision(self) -> bool:
 				return True
 
 		return False
+
+	def game_over(self):
+		"""Function that restarts the game upon game over."""
+
+		self.snake = SnakeGym(self.rows,self.cols, self.make_snake_rand_pos())
+		self.generate_fruit()
+		self.restart = True
+		if self.score > self.high_score:
```

Comment:
is self.high_score being used?

Reply:
Isn't used in the RL stuff, is used in the GA stuff but since we're overriding snakeGame.py I'll just remove it from snakeGameGym.py